functions/Copy-DbaDbQueryStoreOption.ps1

```diff
@@ -147,19 +158,58 @@ function Copy-DbaDbQueryStoreOption {
-                                CollectionInterval  = $SourceQSConfig.StatisticsCollectionIntervalInMinutes;
-                                MaxSize             = $SourceQSConfig.MaxStorageSizeInMB;
-                                CaptureMode         = $SourceQSConfig.QueryCaptureMode;
-                                CleanupMode         = $SourceQSConfig.SizeBasedCleanupMode;
-                                StaleQueryThreshold = $SourceQSConfig.StaleQueryThresholdInDays;
+                            if ($sourceServer.VersionMajor -eq 13) {
+                                $setDbaDbQueryStoreOptionParameters = @{
+
+                                    SqlInstance         = $destinationServer;
+                                    SqlCredential       = $DestinationSqlCredential;
+                                    Database            = $db.name;
+                                    State               = $SourceQSConfig.ActualState;
+                                    FlushInterval       = $SourceQSConfig.DataFlushIntervalInSeconds;
+                                    CollectionInterval  = $SourceQSConfig.StatisticsCollectionIntervalInMinutes;
+                                    MaxSize             = $SourceQSConfig.MaxStorageSizeInMB;
+                                    CaptureMode         = $SourceQSConfig.QueryCaptureMode;
+                                    CleanupMode         = $SourceQSConfig.SizeBasedCleanupMode;
+                                    StaleQueryThreshold = $SourceQSConfig.StaleQueryThresholdInDays;
+                                }
+                            } elseif ($sourceServer.VersionMajor -ge 14) {
+                                $setDbaDbQueryStoreOptionParameters = @{
+
+                                    SqlInstance          = $destinationServer;
+                                    SqlCredential        = $DestinationSqlCredential;
+                                    Database             = $db.name;
+                                    State                = $SourceQSConfig.ActualState;
+                                    FlushInterval        = $SourceQSConfig.DataFlushIntervalInSeconds;
+                                    CollectionInterval   = $SourceQSConfig.StatisticsCollectionIntervalInMinutes;
+                                    MaxSize              = $SourceQSConfig.MaxStorageSizeInMB;
+                                    CaptureMode          = $SourceQSConfig.QueryCaptureMode;
+                                    CleanupMode          = $SourceQSConfig.SizeBasedCleanupMode;
+                                    StaleQueryThreshold  = $SourceQSConfig.StaleQueryThresholdInDays;
+                                    MaxPlansPerQuery     = $QueryStoreOptions.MaxPlansPerQuery;
+                                    WaitStatsCaptureMode = $QueryStoreOptions.WaitStatsCaptureMode;
+                                }
                             }
+                            #For 2019 RTM
```

Comment:
I know it's just comments but I have OCD ;) Please switch to multi-line comments, which are available in PowerShell as `<#`  and `#>`

Reply:
I like OCD :) I can update the test.  Gives me practice.

Reply:
Thanks much, looks good. that latest failure is appveyor being a 🍑 , and not due to your changes. I'll be merging shortly!